bin/compile

```diff
@@ -1 +1,36 @@
 #!/bin/bash
+
+BUILD_DIR=$1
+CACHE_DIR=$2
+LP_DIR=`cd $(dirname $0); cd ..; pwd`
```

Comment:
I still need to figure that out... I lifted this from the `apt` buildpack (along with the later export for other buildpacks) and still need to work out what it does and whether it's necessary or optimal.